aws-cli/README.md

```diff
@@ -0,0 +1,72 @@
+# aws 
+
+This task performs operations on Amazon Web Services resources using `aws`.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/aws-cli/aws-cli.yaml
+```
+
+## Parameters
+
+- **SCRIPT**: The script of aws commands to execute e.g. `aws $1 $2` This will take
+ the first value and second value of ARGS as `s3` and `ls` (default: `aws $@`)
+- **ARGS**: The arguments to pass to `aws` CLI, which are appended 
+    to `aws` e.g. `s3 ls` ( default: `["help"]` ).
+
+
+## Workspace
+
+- **source**: To mount file which is to be uploaded to the aws resources, 
+    this should be mounted using any volume supported in workspace.
+
+
+## Secret
+
+AWS `credentials` and `config` both should be provided in the form of `secret`,
+ which is mounted as `volume` in the task.
+
+[This](./example/secret.yaml) example can be referred to create `aws-secrets`.
```

Comment:
hm this link doesnt work in the preview! this is a super minor thing, but i think if you drop the `.` it'll probably work :D

Reply:
besides this example, do you have any other docs you can link users to with guidance around creating these secrets? (e.g. aws docs)

I find that setting up secrets correctly can be one of the most challenging parts of getting Tasks to work :D

Reply:
for the secrets https://github.com/tektoncd/catalog/pull/287#discussion_r414690879 
and sure will correct the link in the README.

Reply:
> hm this link doesnt work in the preview! this is a super minor thing, but i think if you drop the `.` it'll probably work :D

why wouldn't it ? it's relative to the folder of the file :thinking:

Reply:
Yes, it is working. I cross checked that. 
@bobcatfish can you please check once again? :sweat_smile: